src/SFA.DAS.EmployerFinance/Data/AccountRepository.cs

```diff
@@ -26,25 +28,53 @@ public async Task<string> GetAccountName(long accountId)
 
             parameters.Add("@accountId", accountId, DbType.Int64);
 
-            var result = await _db.Value.Database.Connection.QueryAsync<string>(
+            var result = await _accountDb.Value.Database.Connection.QueryAsync<string>(
                 sql: "SELECT Name FROM [employer_account].[Account] WHERE Id = @accountId",
                 param: parameters,
-                transaction: _db.Value.Database.CurrentTransaction.UnderlyingTransaction,
+                transaction: _accountDb.Value.Database.CurrentTransaction.UnderlyingTransaction,
                 commandType: CommandType.Text);
 
             return result.SingleOrDefault();
         }
 
         public async Task<Dictionary<long, string>> GetAccountNames(IEnumerable<long> accountIds)
         {
-            var result = await _db.Value.Database.Connection.QueryAsync<AccountNameItem>(
+            var result = await _accountDb.Value.Database.Connection.QueryAsync<AccountNameItem>(
                 sql: "SELECT Id, Name FROM [employer_account].[Account] WHERE Id IN @accountIds",
                 param: new { accountIds = accountIds },
-                transaction: _db.Value.Database.CurrentTransaction.UnderlyingTransaction);
+                transaction: _accountDb.Value.Database.CurrentTransaction.UnderlyingTransaction);
```

Comment:
is the idea that after the one time script is ran the _accountDb will be removed completely as a dependency from finance?

Reply:
Yep, there is a story for the data migration.